### ISSUE TITLE
Need a way to .Navigate() to "_top"; break out of "Shell"

### ISSUE THREAD

**Issue by bc3tech** [Windows-XAML/Template10#170]
If your nav paradigm dictates that at some point you no longer have the option to utilize the Hamburger menu, then it's a real pain to try and hide the button and collapse the menu. Especially in a way where the visual state manager doesn't override what you specify.

It would be beneficial, I think, to have a way to call `.Navigate(` with some sort of indicator that this will "break out" of the Shell's content area and come to the "top" of the window. At the same time, using 'back' should get the user back to where they were - inside the shell.

**Comment by darenm** [comment]
So, first and foremost - do remember that our Shell and Hamburger menu implementation is totally optional and you are free to implement your own Shell that handles this behavior. If you do create your own, send us a pull request.

In terms of building something like this, it is important to consider suspend/resume scenarios and navigation serialization. This sort of nested navigation pattern usually requires multiple frames, which would need to be serialized and restored, etc.  To achieve something like you are talking about would also require abstracting the back navigation away from a specific frame into an aggregating service that understands the relationship between Frames and knows when back/forward transitions between Frames.

In short, I agree that this is a perfectly viable approach to building an app, just that I'm not sure this scenario is a high-percentage need and one we would incorporate anytime soon. That said, we do have multiple Windows on the backlog, which could require some of this plumbing to be constructed...

It should be noted that there is a way to inject top level content above the hamburger, etc. in the Shell and we use this to show login UIs, etc.

**Comment by JerryNixon** [comment]
HamburgerMenu.IsFullScreen will get you a similar effect. Is that enough?

**Comment by bc3tech** [comment]
@JerryNixon i'm not sure that's what i'm looking for, but i'll check it out.
@darenm you're completely right when looking at this for a fully robust solution. at this point i'm just scrambling to get _anything_ to work for this scenario. I'll have a look at what the Login UI calls do and see if I can use a similar approach in my app. Thanks.

**Comment by bc3tech** [comment]
Got it pseudo-working in my codebase with a new interface for `Page` types: `IMenuOptionalPage`.
In `HamburgerMenu` control, I now react to the `Navigated` events and process the relevant parameters for the interface showing or hiding the hamburger and/or menu pane accordingly.

If I get time I'll work this in to the Minimal sample and submit a PR.

**Comment by JerryNixon** [comment]
As a Sample, not in Minimal.
